game_api.txt

```diff
@@ -155,7 +155,7 @@ The doors mod allows modders to register custom doors and trapdoors.
 ### Fence gate definition
 
 	description = "Wooden Fence Gate",
-	texture = "default_wood.png",
+	texture = "default_wood.png", -- `backface_culling` will automatically be set to `true` if it's not given.
```

Comment:
shouldn't we have a separate style for documents through? They aren't code so it doesn't make sense to use code style guidelines.